src/signals/settings/categories/Detail/components/CategoryForm/CategoryForm.tsx

```diff
@@ -83,45 +97,41 @@ const CategoryForm = ({ data, history, onCancel, onSubmitForm, readOnly }) => {
 
             <FieldGroup>
               <TextArea
-                defaultValue={data.description}
+                defaultValue={data?.description}
                 disabled={readOnly}
-                hint="Ter verduidelijking van de inhoud van de categorie"
                 id="description"
                 label={<strong>Omschrijving</strong>}
                 name="description"
                 readOnly={readOnly}
-                rows="8"
+                rows={8}
               />
             </FieldGroup>
 
             {responsibleDepartments.length > 0 ? (
               <FieldGroup>
-                <dt><strong>Verantwoordelijke afdeling</strong></dt>
+                <StyledDefinitionTerm><strong>Verantwoordelijke afdeling</strong></StyledDefinitionTerm>
```

Comment:
The FiledGroup is a `div`. I wouldn't use here `dt/dd` unless was a list and I had used dl as parent. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl